debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (native)
```

Comment:
Please make this

```suggestion
3.0 (quilt)
```

Reply:
see comment above. We should also add a watch-file.

Reply:
https://salsa.debian.org/python-team/packages/key-mapper/-/blob/master/debian/watch works for me ;-).